applications/graphQL/tests/test_query.py

```
@@ -0,0 +1,120 @@
+        test_cases = [(1, True), (999, False)]  # Existing insult  # Non-existent insult
+
+        for insult_id, should_exist in test_cases:
+            query = (
+                """
+                query {
+                    insultById(id: %s) {
+                        content
+                    }
+                }
+            """
+                % insult_id
```

Ruff (0.12.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)